docs/digital/cirq_interop/squin_to_cirq.md

```diff
@@ -0,0 +1,59 @@
+# Converting squin to cirq
+
+You can convert a squin kernel function to a `cirq.Circuit` object.
+The output circuit will feature gates that most closely resemble the kernel you put in.
+
+## Basic usage
+
+You can obtain a circuit using the `squin.cirq.emit_circuit` function.
+
+```python
+from bloqade import squin
+
+@squin.kernel
+def main():
+    q = squin.qubit.new(2)
+    h = squin.op.h()
+    squin.qubit.apply(h, q[0])
+    cx = squin.op.cx()
+    squin.qubit.apply(cx, q[0], q[1])
+    squin.qubit.measure(q)
+
+circuit = squin.cirq.emit_circuit(main)
+print(circuit)
+```
+
+There is one crucial difference between a squin kernel and a cirq circuit:
+the qubits are defined inside a kernel, whereas they are defined outside of a circuit.
```

Comment:
```suggestion
the qubits are defined inside a kernel, whereas for a circuit they are defined outside.
```
I assume this is what you mean but feel free to override me if this isn't correct!